app/Http/Requests/RegisterUser.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Requests;
+use Illuminate\Contracts\Validation\Validator;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Foundation\Http\FormRequest;
+
+class RegisterUser extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'email' => 'required|string|email|max:255|unique:users',
+            'password' => 'required|string|min:6|confirmed'
+        ];
+    }
+    
+    protected function failedValidation(Validator $validator) {
```

Comment:
ovo mislim da ti ne treba, on ce sam da baci exception za to ako ne prodje validaciju